layers/+source-control/git/packages.el

```diff
@@ -234,6 +234,9 @@
       ;; whitespace
       (define-key magit-status-mode-map (kbd "C-S-w")
         'spacemacs/magit-toggle-whitespace)
+      ;; https://magit.vc/manual/magit/MacOS-Performance.html
+      (when (eq system-type 'darwin)
+        (setq magit-git-executable (s-trim (shell-command-to-string "which git"))))
```

Comment:
I'm probably nitpicking but going through shell raises an eyebrow, and in the odd edge case of not having git setting the executable to empty string(?) feels slightly wrong. I would've thought something like this, but I'm not sure if deferring to shell would actually be preferrable
```elisp
(when-let ((git (executable-find "git")))
  (setq magit-git-executable git))
```
I suppose one could argue this should be safe on all OSs and there might be some benefit for being consistent, but :shrug:

Reply:
I tend to agree to @tko, this code should be fairly save for all OSes so for the sake of consistency please remove the `only darwin when` please.

Reply:
@tko @smile13241324 alright, removed the condition in ad941de